core/geometry/src/test/curve/Arc3d.test.ts

```diff
@@ -717,6 +719,60 @@ describe("Arc3d", () => {
     GeometryCoreTestIO.saveGeometry(allGeometry, "Arc3d", "createCircularStartTangentEnd");
     expect(ck.getNumErrors()).equals(0);
   });
+  it.only("createCircularStartEndRadius", () => {
```

Comment:
`only` and `true,true` should be removed.

Reply:
Ack! Good catch.